.github/workflows/ci-linux.yml

```diff
@@ -66,7 +66,9 @@ jobs:
 
     - name: Package binaries
       if: (github.event_name == 'release')
-      run: zip alr-bin-linux.zip bin/alr LICENSE.txt
+      run: |
+        echo 'sudo cp -r bin share /usr/local/' > INSTALL.md
```

Comment:
I'd rather have a proper INSTALL.md file a bit more fleshed out, where this command is not just given but also explained. Some people may prefer to use another prefix not requiring sudo.

Reply:
Should I make this Linux-specific or even distro specific? I believe that where to put Bash completion files is a bit distro-dependent.

Reply:
Can this be found automatically? Otherwise I don't think we want to manually maintain such a thing. We can explain in the INSTALL.md file that the given structure is for Debian and should be tweaked if needed. Also yes, I think the INSTALL.md should make clear this is only for Unix-like, and that there's a Windows installer.